monkestation/strings/random_crimes.txt

```diff
@@ -59,3 +59,16 @@ Reached Skate Nirvana, Flubbed Up
 Too Cool
 Cooking Reality-Bending Food
 Recycling
+Caused a Intergalactic War Between NanoTrasen And The Clown Union, Due To Adding Silentium To a Banana
+Stun Prodded a Clown And Spacing Said Clown Out a Airlock
```

Comment:
This one might need to go. They're meant to be obviously fake crimes when looked into.

Reply:
which one, the stun prodded clown one?

Reply:
Yes

Reply:
aight will get to removing that one

Reply:
removed the stun prodded clown one